google/cloud/completion_queue_test.cc

```diff
@@ -404,7 +405,11 @@ TEST(CompletionQueueTest, MakeRpcsAfterShutdown) {
           },
           r1, std::make_unique<grpc::ClientContext>())
           .then([](future<StatusOr<Response>> f) {
-            EXPECT_EQ(StatusCode::kCancelled, f.get().status().code());
+            auto status = f.get().status();
+            EXPECT_EQ(StatusCode::kCancelled, status.code());
```

Comment:
Nit: Consider `EXPECT_THAT(status, StatusIs(StatusCode::kCancelled));` for a better error message.

Reply:
done